### ISSUE TITLE
Remote Control: Button pressed trigger?

### ISSUE THREAD

**Issue by tumidinoda** [morzzz007/node-tradfri#9]
How can I trigger a button pressed event?

**Comment by morzzz007** [comment]
Hi, can you elaborate further on what do you mean by a button pressed event? You want to trigger a button press on the remote from node?

**Comment by tumidinoda** [comment]
Yes, having an IKEA remote control. I warnt to trigger on/off, lighter/darker and color button pressed events.
